conftest.py

```diff
@@ -13,6 +13,8 @@
 DB_FILE_PATH = f"{HERE}/jupyter_scheduler/tests/testdb.sqlite"
 DB_URL = f"sqlite:///{DB_FILE_PATH}"
 
+TEST_ROOT_DIR = f"{HERE}/jupyter_scheduler/tests/test_root_dir"
+
```

Comment:
1. `os.path.join()` should be used to construct file paths, as the directory delimiter can be platform-dependent.
    - Windows uses backslashes instead of forward slashes, e.g. `C:\some_dir\child_dir`.
2. When using directories in tests, we should not commit the test dirs into the source repo, as this doesn't scale well when more tests are added. Instead, you should use the [`tmp_path` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) provided by `pytest`. When used, this will create a new temporary directory (under `/tmp` on Linux) and return the path to that temporary directory. 

    So here you would want to replace `TEST_ROOT_DIR` with something like:

    ```python
    @pytest.fixture
    def scheduler_root_dir(tmp_path):
        return os.path.join(tmp_path, 'workspace_root')
    ```

For `DB_FILE_PATH`, you should use the `jp_data_dir` fixture provided by `jupyter_server.pytest_plugin`. This fixture is already under `tmp_path`.